python/tvm/relay/frontend/tensorflow.py

```diff
@@ -3682,6 +3729,30 @@ def from_tensorflow(graph, layout="NHWC", shape=None, outputs=None):
     params : dict of str to tvm.nd.NDArray
         Dict of converted parameters stored in tvm.nd.NDArray format
     """
+    from tensorflow.python.eager.function import ConcreteFunction
+
+    if isinstance(graph, ConcreteFunction):
+        try:
+            from tensorflow.python.framework import convert_to_constants
+            from tensorflow.core.protobuf import config_pb2
+        except ImportError as e:
+            raise ImportError("Unable to import tensorflow which is required {}".format(e))
+        concrete_func = graph
+        graph = convert_to_constants.convert_variables_to_constants_v2(concrete_func).graph
+        signature = _build_signature_def(graph, concrete_func.inputs, concrete_func.outputs)
+        graph_def = graph.as_graph_def()
+
+        # Some optimization
+        config = config_pb2.ConfigProto()
+        rewriter_config = config.graph_options.rewrite_options
+        rewriter_config.optimizers[:] = [
+            "debug_stripper",
+            "arithmetic",
+            "dependency",
+            "arithmetic",
```

Comment:
typo: duplicate of line 3750

Reply:
"arithmetic" is not a duplicate, this transform happen again.
The transforms order was  with a ref. from TensorflowJS project. This can be revised later as needed.